collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -778,21 +787,210 @@ def process_dialogflow(
         slot_labels)
 
     train_queries, train_slots, test_queries, test_slots = \
-        partition_df_data(intent_queries, slot_tags, split=dev_split)
+        partition_data(intent_queries, slot_tags, split=dev_split)
 
-    write_df_files(train_queries, f'{outfold}/train.tsv')
-    write_df_files(train_slots, f'{outfold}/train_slots.tsv')
+    write_files(train_queries, f'{outfold}/train.tsv')
+    write_files(train_slots, f'{outfold}/train_slots.tsv')
 
-    write_df_files(test_queries, f'{outfold}/test.tsv')
-    write_df_files(test_slots, f'{outfold}/test_slots.tsv')
+    write_files(test_queries, f'{outfold}/test.tsv')
+    write_files(test_slots, f'{outfold}/test_slots.tsv')
 
-    write_df_files(slot_labels, f'{outfold}/dict.slots.csv')
-    write_df_files(intent_names, f'{outfold}/dict.intents.csv')
+    write_files(slot_labels, f'{outfold}/dict.slots.csv')
+    write_files(intent_names, f'{outfold}/dict.intents.csv')
 
     return outfold
 
 
-def write_df_files(data, outfile):
+def readCSV(file_path):
```

Comment:
you might want to use pandas' read_csv https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html

Reply:
won't be using any of the pandas features so it may not be so beneficial bringing it in here. Moreover, pandas would provide perf gain on large amount of data. We are probably dealing with smaller datasets within dialog for now, especially from mturk.

Reply:
makes sense. can you rename the function to read_csv to be consistent with the python naming convention?

Reply:
done.